reference/array/functions/array-diff.xml

```diff
@@ -93,18 +104,54 @@ Array
     </screen>
    </example>
   </para>
+
+  <para>
+   <example>
+    <title><function>array_diff</function> przykład z niepasującymi typami</title>
```

Comment:
To tłumaczenie wydaje się być bardzo skalkowane gramatycznie z angielskiego. Dałbym albo klasycznie: "Przykład użycia array_diff z niepasującymi typami" lub "array_diff z niepasującymi typami", jako że przy renderowaniu to i tak będzie miało pełną postać: "Przykład #1: `array_diff()` z niepasującymi danymi"